packages/cape/src/encrypt.ts

```diff
@@ -39,11 +39,22 @@ export async function encrypt(plainText: Uint8Array, publicKey: Uint8Array): Pro
  *
  * @param plainText The plain text input to encrypt.
  */
-export async function aesEncrypt(plainText: Uint8Array): Promise<EncryptResponse> {
-  // Generate a new key
-  const key = forge.random.getBytesSync(32);
+export async function aesEncrypt(plainText: Uint8Array, key?: string): Promise<EncryptResponse> {
+  let useKey: string;
+  // Generate a new key if the key isn't provided.
```

Comment:
why would we want to generate a key if the key isn't provided?

Reply:
Oh nevermind I see this is for the envelop encryption. Guess my question is the opposite. Why would we ever want to pass an key into here?

Reply:
Gavin mentioned that being able to specify the AESkey might be good for the heatmap demo.

Reply:
Yeah going to need to understand that more. I don't really see why we'd ever need to specify an aes key because the encrypted version ends up just getting bundled with the encrypted data anyway.

Reply:
That would be good, we can just get rid of this, this is the part of the code that's not as well tested anyway.